<launch>
    <arg name="fps" default="10" />
    <arg name="model" default="$(find neuro_car)/models/eyes.model" />
    <arg name="mode" default="eyes_alpha" />

    <node name="svm_eyes" pkg="neuro_car" type="svm" output="screen">
        <param name="fps" type="int" value="$(arg fps)" />
        <param name="model" type="string" value="$(arg model)" />
        <param name="mode" type="string" value="$(arg mode)" />
    </node>
</launch>
